salt/orchestrate/aws/cloud_profiles/backup_host.conf

```diff
@@ -15,9 +16,8 @@ backup_host:
     - DeviceName: /dev/xvda
       Ebs.VolumeSize: 8
       Ebs.VolumeType: gp2
-    - DeviceName: /dev/xvdb
-      Ebs.VolumeSize: 400
-      Ebs.VolumeType: gp2
+    - device: /dev/xvdb
+      volume_id: vol-0bd5c579e0336bcef
```

Comment:
Rather than hardcoding the volume ID it would be better to use a labelled volume that we mount and unmount as part of the orchestrate state. If the volume is launched as part of the profile I would be worried about it being terminated along with the instance...